packages/ua-devtools-evm/src/oapp/sdk.ts

```diff
@@ -138,6 +138,22 @@ export class OApp extends Ownable implements IOApp {
         }
     }
 
+    async setCallerBpsCap(callerBpsCap: bigint): Promise<OmniTransaction> {
+        const data = this.contract.contract.interface.encodeFunctionData('setCallerBpsCap', [callerBpsCap])
+
+        return {
+            ...this.createTransaction(data),
+            description: `Setting caller BPS cap to ${callerBpsCap}`,
+        }
+    }
+
+    @AsyncRetriable()
+    async getCallerBpsCap(): Promise<bigint> {
+        const callerBpsCap = await this.contract.contract.callerBpsCap()
+
+        return BigNumberishBigIntSchema.parse(callerBpsCap)
+    }
```

Comment:
As mentioned in another comment on this PR, not all contracts will have these functions (only `OFTWrapper` or its children will) so perhaps we should wrap the logic within these functions in a try-catch?

For testing purposes in this PR, I added these getters/setters to `DefaultOApp`

Reply:
I think we have an example of handling this is stargate v2 - in asset sdk i believe, look for lpToken getter

Reply:
ah gotcha, this is exactly what I was looking for thanks!